ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -289,7 +294,11 @@ private void updateClippingToRect(Rect clippingRect) {
 
   private void updateSubviewClipStatus(Rect clippingRect, int idx, int clippedSoFar) {
     View child = Assertions.assertNotNull(mAllChildren)[idx];
-    sHelperRect.set(child.getLeft(), child.getTop(), child.getRight(), child.getBottom());
+    sHelperRect.set(
+      child.getLeft() + (int) child.getTranslationX(),
```

Comment:
Updated to do like AOSP, use Matrix#mapRect and take a sligtly larger rect when converting from a float rect to a int rect. This make sure that float -> int doesn't remove views that may still be visible.

https://github.com/android/platform_frameworks_base/blob/4535e11fb7010f2b104d3f8b3954407b9f330e0f/core/java/android/view/ViewGroup.java#L5049